spk/rdiff-backup/Makefile

```diff
@@ -18,7 +18,7 @@ MAINTAINER = SynoCommunity
 DESCRIPTION = Reverse differential backup tool, over a network or locally.
 STARTABLE = no
 DISPLAY_NAME = rdiff-backup
-CHANGELOG = "1. Update to v2.2.6.<br/>2. Bring back support for ARMv5.<br/>3. Use alternate tmp dir."
+CHANGELOG = "1. Update to v2.2.6.<br/>2. Bring back support for ARMv5.<br/>3. Use alternate tmp dir.<br/>4. Migrate to Python 3.12."
```

Comment:
ARMv5 is not supported anymore since python312 is not supported.

Reply:
Removed